README.md

```diff
@@ -12,7 +12,7 @@ This repository is broken up into:
   * [billing](/scripts/billing) - Example queries over the GCP billing export
 * [UDFs](/udfs) - User-defined functions for common usage as well as migration
   * [community](/udfs/community) - Community contributed user-defined functions
-  * [teradata](/udfs/migration/teradata) - UDFs which mimic the behavior of proprietary functions in Teradata
```

Comment:
My intention was to list out all the source systems under `/udfs/migration` so that a user landing on the main README would know what exists in the subdirectories without having to drill-down. Let me know if that doesn't make sense.

Reply:
Presently README mentions only Teradata but there are more databases under `migration`. Probably we should list all source databases on the main page?

Reply:
Yeah that makes sense to me.